### ISSUE TITLE
sqlite_scanner error in CI

### ISSUE THREAD

**Issue by yafimvo** [ploomber/jupysql#173]
The CI process may encounter the following error and fail during execution:

```
_________________________ test_boxplot_stats_exception _________________________

chinook_db = '/Users/runner/work/jupysql/jupysql/src/tests/tmp/my.db'

    def test_boxplot_stats_exception(chinook_db):
        con = duckdb.connect(database=":memory:")
>       con.execute("INSTALL 'sqlite_scanner';")
E       duckdb.IOException: IO Error: Failed to download extension "sqlite_scanner" at URL "http://extensions.duckdb.org/v0.7.1/osx_amd64/sqlite_scanner.duckdb_extension.gz"
E       Extension "sqlite_scanner" is an existing extension.
E       
E       Are you using a development build? In this case, extensions might not (yet) be uploaded.
```

We need to re-run the CI several times to make it pass.

Full error log: [here](https://github.com/ploomber/jupysql/actions/runs/4284200029/jobs/7460921972#step:6:42)

cc: @edublancas @idomic

**Comment by edublancas** [comment]
maybe this was an intermittent issue due to the newly released duckdb 0.7.1?

I pushed today to master and the CI passed: https://github.com/ploomber/jupysql/commit/ecd91c1a598a2c9f1beeecf5c17474795b64a574

**Comment by yafimvo** [comment]
yes, the latest version works well. we can close it.
